p11-kit/uri.c

```diff
@@ -1393,17 +1393,22 @@ p11_kit_uri_parse (const char *string, P11KitUriType uri_type,
 	uri->pin_value = NULL;
 	uri->slot_id = (CK_SLOT_ID)-1;
 
+	/* Parse the path. */
```

Comment:
Well, I can't really tell how to merge them :(
i basically split it because it started to look bad to do a lot of special casing there.